docs/cugraph/source/api_docs/cugraph-ops/index.rst

```diff
@@ -0,0 +1,16 @@
+=========================
+cugraph-ops API reference
+=========================
+
+This page provides a list of all publicly accessible modules, methods and classes through `pylibcugraphops.*` namespace.
+
+.. toctree::
+    :maxdepth: 2
+    :caption: API Documentation
+
+    graph_types
```

Comment:
should also add pytorch here at the top IMO (just noticed that our own docs have them at the end, but pytorch is the main API for end-users so this should be the first thing to see)

```suggestion
    pytorch
    graph_types
```

Reply:
Adding the C++ API is an upcoming step.   Working on covering from Doxogen to Sphinx.  Once I have the process setup for the other cugraph C code, then I'll be rolling in cugraph-ops.

Reply:
I believe you now added the pytorch entry twice, as well as not adding it at the top as suggested above.
Let's just remove one of these instances, or completely remove this if it doesn't pass CI